src/index.ts

```diff
@@ -129,3 +129,50 @@ export * from './matrix/matrixPQN';
 export * from './matrix/matrixZRescale';
 
 export * from './utils/createSequentialArray';
+
+export type ArrayType = number[] | Float64Array | Float32Array | Uint16Array;
+export interface Zone {
+  fromIndex?: number;
+  toIndex?: number;
+  to: number;
+  nbPoints?: number;
+  from: number;
+}
+
+export interface Point {
+  x: number;
+  y: number;
+  index?: number;
+}
+export interface XYPoints {
```

Comment:
Should be renamed to DataXY (that is already defined higher but should be replaced)

Reply:
Resolved